web-common/src/runtime-client/local-service.ts

```diff
@@ -0,0 +1,163 @@
+import type { PartialMessage } from "@bufbuild/protobuf";
+import { createPromiseClient } from "@connectrpc/connect";
+import { createConnectTransport } from "@connectrpc/connect-web";
+import type {
+  CreateMutationOptions,
+  CreateQueryOptions,
+} from "@rilldata/svelte-query";
```

Comment:
Shouldn't it use `@tanstack/svelte-query`? That is used below for `import { createMutation, createQuery }`